Code/Tools/FBuild/FBuildCore/Helpers/ToolManifest.cpp

```diff
@@ -117,6 +120,11 @@ bool ToolManifest::Generate( const Node * mainExecutable, const Dependencies & d
         *pos = xxHash::Calc32( relativePath );
         ++pos;
     }
+	for (size_t i = 0; i < numEnvVars; ++i)
+	{
+		*pos = xxHash::Calc32(m_CustomEnvironmentVariables[i]);
+		++pos;
+	}
```

Comment:
Yes, you're correct, and removing them from the hash would allow much better toolchain re-use.